src/Symfony/Component/Form/Extension/Core/Type/FormType.php

```diff
@@ -146,6 +146,14 @@ public function configureOptions(OptionsResolver $resolver)
             };
         };
 
+        // Derive "upload_max_size_message" closure from "post_max_size_message" option,
+        // to allow override or translate the message only when it's needed and not during the resolve `options`.
```

Comment:
I think there are typo: `overriding` and `translating`.

What about something like:
```php
// Wrap "post_max_size_message" in a closure to translate it lazily
 $uploadMaxSizeMessage ...
```